core/db/entities.py

```diff
@@ -12,6 +12,20 @@
 from core.db.types import GUID
 from core.util import get_itl_regions_from_postcodes
 
+project_geospatial_association = db.Table(
+    "project_geospatial_association",
+    db.Column(
+        "project_id", GUID(), db.ForeignKey("project_dim.id", ondelete="CASCADE"), nullable=False, primary_key=True
+    ),
+    db.Column(
+        "geospatial_id",
+        GUID(),
+        db.ForeignKey("geospatial_dim.id", ondelete="RESTRICT"),
+        nullable=False,
+        primary_key=True,
+    ),
+)
+
```

Comment:
Mainly because it wasn't a proper Model class like the others in the file so felt odd to stick it amongst the rest - but happy to move it to sit alphabetically in the list/sit it next to projects or geospatial if that's better practice?

Reply:
It's nice to co-locate things together, I think.

Reply:
Have co-located to above `GeospatialDim` - the association table needs to be defined before it can be used in either `GeospatialDim` or `Project`.